drivers/scheduler/k8s/specs/postgres-backup-psa-restricted/pxd/px-storage-class.yaml

```diff
@@ -0,0 +1,10 @@
+##### Portworx storage class
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: postgres-sc
+provisioner: kubernetes.io/portworx-volume
```

Comment:
Same here.. change with CSI driver

Reply:
done

Reply:
Caution: Torpedo, by default, uses the `kubernetes.io/portworx-volume` provisioner. We need to pass `CSI_APP_LIST` for it to deploy apps with the `pxd.portworx.com` provisioner. Cc: @sn-px